docs/overview/what-is-backstage.md

```diff
@@ -15,15 +15,15 @@ to create a streamlined development environment from end to end.
 
 Out of the box, Backstage includes:
 
-- [Backstage Service Catalog](/docs/features/software-catalog/index.md) for
+- [Backstage Service Catalog](/docs/features/software-catalog/index) for
```

Comment:
This 404s in GitHub. Is it possible to make docusaurus support .md links?

Reply:
Good call, found a better way to fix it